doc/classes/DisplayServer.xml

```diff
@@ -1161,6 +1161,14 @@
 				Returns the screen's size in pixels. See also [method screen_get_position] and [method screen_get_usable_rect].
 			</description>
 		</method>
+		<method name="screen_get_subpixel_layout" qualifiers="const">
+			<return type="int" enum="DisplayServer.ScreenSubpixelLayout" />
+			<param index="0" name="screen" type="int" default="-1" />
+			<description>
+				Returns sub-pixel structure of the screen.
```

Comment:
Does the value returned here depend on screen orientation? I'd mention this in the documentation.